core/src/test/java/com/expedia/aquila/core/util/MathUtilTest.java

```diff
@@ -0,0 +1,62 @@
+package com.expedia.aquila.core.util;
+
+import org.junit.rules.ExpectedException;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+public class MathUtilTest {
+
+    @Rule public final ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void incrAndLog() {
+      final double[] data = {1.0};
```

Comment:
Can this be inlined (as long as it fits on a line)?

Reply:
Yes. Only the final test won't fit on a line.